backend/app/routers/experiment_templates.py

```diff
@@ -20,24 +20,32 @@
 router = APIRouter()
 
 
+class ExperimentTemplateFilter:
```

Comment:
I suppose I'd change this class so that it would inherit it from `pydantic.BaseModel` class, in which case we wouldn't need to define the constructor, but rather define the attributes themselves (this has been done with `Pagination` as well)

Reply:
Thanks! I've totally missed this..